api/addr.go

```diff
@@ -0,0 +1,16 @@
+package api
+
+const (
+	// GetAddrsPath is the URL path to fetch a list of public nodes
+	GetAddrsPath = "/api/addrs"
+
+	IPVersion       = "ipVersion"
+	ServiceFlag     = "serviceFlag"
+	ProtocolVersion = "protocolVersion"
```

Comment:
davec suggested that we rename these to the same fields as in the json response ("services" and "pver") and I agree it would make the API more regular.